eleventy.config.js

```diff
@@ -50,6 +50,10 @@ export default function eleventy(eleventyConfig) {
   eleventyConfig.addPassthroughCopy({ "node_modules/@github/relative-time-element/dist": "assets/scripts/relative-time/" });
   eleventyConfig.addPassthroughCopy({ "node_modules/@zachleat/filter-container/filter-container.js": "assets/scripts/filter-container.js" });
 
+  eleventyConfig.addFilter("autolink", function (content) {
+    return content.replaceAll(/(?<![\/>])\b([A-Z]+-[0-9]+)\b/g, '<a href="/browse/$1/">$1</a>');
```

Comment:
There are some false positives for when the issue number is passed as a query parameter in a URL. This is for those `icft` cases. You can see an example on FLUID-6148 and ENGAGE-124. I think you just need to ignore the a leading `=` as well.